docs/_sources/providers.rst.txt

```diff
@@ -118,6 +118,59 @@ Contentful
 
 Uses client :ref:`Contentful <client_contentful>`.
 
+Qualys
+======
+
+Provides password rotation support for Qualys accounts. Only `rotate` has been implemented,
+`distribution` is not supported.
+
+For a given secet specification, the `rotate` function fetches the contents of a AWS Secrets Manager
+secret located at `keydra/qualys/<key>` (using the key from the spec), and changes the password of the user
+specified. As Qualys does not permit a user changing their own password via API, the Secret needs to specify a
+second account to be used to make the change.
+
+For example, for a secret spec of:
+
+.. code-block:: yaml
+
+    sample:
+      description: API User
+      key: api
+      provider: qualys
+      rotate: nightly
+      distribute:
+      -
+         key: keydra/qualys/api
+         provider: secretsmanager
+         source: secret
+         envs:
+            - prod
+
+The provider will take an AWS Secrets Manager secret, located at `keydra/qualys/api`:
+
+.. code-block:: json
+
+   {
+      "platform": "US3",
+      "username": "apiuser",
+      "password": "Ssh.Secret!",
+      "rotatewith": "keydra/qualys/backup"
```

Comment:
Hmm do you reckon it fit better as a provider config item instead?

Reply:
Good idea! Will update.

Reply:
Updated!